src/clib/pio_file.c

```diff
@@ -117,7 +117,7 @@ int PIOc_createfile(const int iosysid, int *ncidp, int *iotype,
     file->buffer.data = NULL;
     file->buffer.next = NULL;
     file->buffer.vid = NULL;
-    file->buffer.ioid = -1;
+    file->buffer.ioid = 0;
```

Comment:
I didn't intend for this to be part of the PR. I was indeed testing it out, but decided that I wanted to stick to comment so real code changes could be reviewed more carefully.